src/index.html

```diff
@@ -12,7 +13,40 @@
       href="styles/index.scss"
```

Comment:
You have linked a SCSS file in your HTML. Browsers can't interpret SCSS, they only understand CSS. Make sure to compile your SCSS to CSS and link the compiled CSS file here.